roles/openshift_logging_mux/defaults/main.yml

```diff
@@ -57,11 +58,11 @@ openshift_logging_mux_file_buffer_storage_type: "emptydir"
 openshift_logging_mux_file_buffer_pvc_name: "logging-mux-pvc"
 
 # required if the PVC does not already exist
-openshift_logging_mux_file_buffer_pvc_size: 4Gi
+openshift_logging_mux_file_buffer_pvc_size: 512Mi
```

Comment:
Lowering the PVC size to be on the same order as the lowered memory limit.

Reply:
Probably not a big deal, but this size PVC will preclude using IOPS-provisioned EBS storage for file buffer storage.  io1 storage has a minimum size of 4Gi.  But users can change this in the Ansible vars if there default storage class is io1.

Reply:
@mffiedler I know @portante arrived at these numbers based upon the mismatch between disk space and the various fluentd queue and buffer settings.  I think we need to figure out how users set one value and it is used to set a few other settings to arrive at the set that is not mismatched

Reply:
@mffiedler, the default PVC size of 1 Gi seems reasonable to have for gp2 storage class.  I'll update the commit.